### PR TITLE
Feature schedule maintenance

### PR DESCRIPTION
**Pull request by tyleryy**
### Overview

Put Maintenance Image temporarily on schedule page until it is developed, for early site release.

- left both images in for comparison as to which is better
- __maybe we can ask graphics to make a maintenance Petr?__

- we could also create an `error.tsx` file and have that render instead? I don't know if this is considered best practice or not.

### CODE REVIEW

**Review by taesungh — COMMENTED** [review]
Do these images have a free license? Would prefer to stick with in-house graphics when possible. Additionally, prior convention is leaving out the schedule page entirely from Navigation, but open to having such a "coming soon" page if not too out of character.

**Comment by tyleryy** [comment]
Yea they have free license to use. I'll send it over slack @taesungh

**Comment by tyleryy** [comment]
image dimensions can be changed if fit

File: apps/site/src/assets/images/maintenance.png
Comment (taesungh) [inline apps/site/src/assets/images/maintenance.png:1]: Chore: asset unused.

**Comment by samderanova** [comment]
We'll be utilizing environment variables to control whether we show the resources and schedule pages or not.

cc: @taesungh

**Comment by taesungh** [comment]
Working on it.

**Comment by taesungh** [comment]
Will need to add values for `MAINTENANCE_MODE_SCHEDULE` and `MAINTENANCE_MODE_RESOURCES` to the **Production** environment in Vercel to show maintenance until we are ready to release.

**Review by samderanova — APPROVED** [review]
Looks good, thanks for working on this. I also added the variables to Vercel.

**Review by taesungh — APPROVED** [review]
Nice, let's squash merge.

**Pull request merged**